apps/modernization-ui/src/components/SideNavigation/SideNavigation.tsx

```diff
@@ -32,7 +32,7 @@ const SideNavigation = ({ title, className, children = [] }: SideNavigationProps
 
     return (
         <div className={classNames('navigation', className)}>
-            <h2>{title}</h2>
+            <h2 className="margin-0">{title}</h2>
```

Comment:
issue: The [navigation class](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/components/SideNavigation/SideNavigation.scss) is already setting styles for an `h2`, that would be the place to apply margins to this specific header.